src/dronevis/tracking/human_tracking.py

```diff
@@ -0,0 +1,165 @@
+            print("Downloading Faster YOLO v4 weights...")
+            yolo_weights_path, headers = urllib.request.urlretrieve(
+                "https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v3_optimal/yolov4.weights", 
+                filename= os.path.join(os.getcwd(),"yolov4.weights"))
+        print("Loading Faster YOLO v4 model ...")
+        self.model = cv2.dnn.readNet(yolo_cfg_path, yolo_weights_path)
+        self.tracker = DeepSort()
+
+
+    def transform_img(self, img: np.ndarray) -> torch.Tensor:
+        """Transform image to tensor
+
+        Args:
+            img (numpy.ndarray): input array
+
+        Returns:
+            torch.Tensor: tensor img
+        """
+        assert self.transform is not None, "Model not initialized. You need to load the model first. Please run `load_model`."
+        return self.transform(to_pil_image(img)).to(self.device)
+
+    def predict(self,image:np.ndarray) -> list:
+        """
+        Detect Humans in an image using YOLO v4
+        Args:
+            image (np.ndarray): input image
+        Returns:
+            List of Detections, where each detection is represented as a tuple
+            of bounding box coordinates, classid and score
+        """
+        detections = []
+        model = cv2.dnn_DetectionModel(self.model)
+        model.setInputParams(size=(416, 416), scale=1/255, swapRB=True)
+        classes, scores, boxes = model.detect(image, self.CONFIDENCE_THRESHOLD, self.NMS_THRESHOLD)
+        for (classid, score, box) in zip(classes, scores, boxes):
+            label = self.class_names[classid[0]]
+            if (label == "person"):
+                detections.append((box,classid,score))
+        return detections
+    
+
+    def detect_webcam(self,video_index: Union[str, int] = 0, window_name: str = "Cam Detection") -> None:
+        """Start webcam detection from video_index
+        *(to quit running this function press 'q')*
+        Args:
+            video_index (Union[str, int], optional): index of video stream device. Defaults to 0 (webcam).
+            window_name (str, optional): name of cv2 window. Defaults to "Cam Detection".
+        """
+        cap = cv2.VideoCapture(video_index)
+        prev_time = 0
+        while True:
+            _, frame = cap.read()
+            detections = self.predict(frame)
+            # Update object tracks with DeepSORT
+            Flag = True
+            if (len(detections) == 0):
+                Flag = False
+            if Flag == True:
+                tracks = self.tracker.update_tracks(detections, frame=frame)
+                # Visualize object tracks
+                for track in tracks:
+                    bbox = track.to_tlbr()
+                    cv2.rectangle(frame, (int(bbox[0]), int(bbox[1])), (int(bbox[2]), int(bbox[3])), (255, 255, 255), 2)
+                    cv2.putText(frame, str(track.track_id), (int(bbox[0]), int(bbox[1]) - 10), 0, 5e-3 * 200, (0, 255, 0), 2)

```

Comment:
Consider using argument names, e.g. :
```python
cv2.putText(
    img=frame,
    text=str(track.track_id),
    org=(int(bbox[0]), int(bbox[1]) - 10),
    fontFace=0, 
    fontScale=5e-3 * 200,
    color=(0, 255, 0),
    thickness=2,
)
```
so every value would be imply its meaning.